### DISCUSSION TITLE
Can zed be used as an embedded code editor in my gui application?

### DISCUSSION THREAD

**Discussion by AngeloChecked** [zed-industries/zed#7628] [Feature Requests]
can I use zed as a library? It's possible? from where can I start?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
